.github/scripts/docker-create-test copy.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+################################################################################
+# Help                                                                         #
+################################################################################
+Help() {
```

Comment:
hmmm maybe we can print at the beginnign of the no-argument run what this will do? (E.g. create local test container)